.changeset/lazy-nails-guess.md

```diff
@@ -0,0 +1,5 @@
+---
+'@firebase/firestore': patch
+---
+
+Add TransactionOptions to runTransaction
```

Comment:
Can you add backticks to the 2 API names and add "param" after TransactionOptions?

Reply:
Done